Amazon.QLDB.Driver/utils/QldbHash.cs

```diff
@@ -67,12 +67,8 @@ public override string ToString()
         /// <returns>Hashed result.</returns>
         internal static QldbHash ToQldbHash(string value)
         {
-            if (value == null)
-            {
-                return ToQldbHash(ValueFactory.NewString(string.Empty));
-            }
-
-            return ToQldbHash(ValueFactory.NewString(value));
+            return value == null ? ToQldbHash(ValueFactory.NewString(string.Empty)) :
```

Comment:
What about:
`return ToQldbHash(ValueFactory.NewString(value ?? string.Empty));`

Reply:
That's a lot better, done.